src/server/requirements.txt

```diff
@@ -7,11 +7,13 @@ python-engineio==3.11.*
 python-socketio==4.4.*
```

Comment:
Should this be an add-on file instead? So the user would always run the `core` and then optionally run the `requirements-pi` if needed? A little extra work for the user but would prevent needing to update this in multiple places.

Reply:
I expect very few users to do this on something other than the Pi, so I'd rather not disturb that use case.